### DISCUSSION TITLE
use PKsim or Mobi to establish PBPK model, with the purpose of predicting brain tissue concentration through blood concentration of the two drugs to optimize the drug administration plan

### DISCUSSION THREAD

**Discussion by pearlzhuzhu** [Open-Systems-Pharmacology/Forum#1540] [Questions & Problems]
Help! I have two small-molecule antibiotics, meropenem and sulbactam, that combine to treat Acinetobacter baumannii. I want to use PKsim or Mobi to establish PBPK model, with the purpose of predicting brain tissue concentration through blood concentration of the two drugs to optimize the drug administration plan. What should the modeling steps be like? Is there any modeling material I can refer to? Hope to get answers! Best wishes!

**Comment by tobiasK2001** [comment]
Hi,

it seems like you are on a starting point for an interesting journey. 
Did you check out this paper: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5080648/
You may also want to check out our tutorials here: 
https://github.com/Open-Systems-Pharmacology/OSP-based-publications-and-content/labels/Tutorial
And finally if you are looking for a F2F hands on Workshop you might want to check out this:
https://lnkd.in/gwA7rYPg

Good luck!

Tobias

**Comment by StephanSchaller** [comment]
Hi @pearlzhuzhu, in additiona to the links from @tobiasK2001, if you are from academia, you can find free well-guided up-to-date online training/tutorials here:
https://learn-esqlabs.talentlms.com/

For industry, this will require some sponsoring:
https://esqlabs.shop/  --> "courses (latest OSP Version)"
